src/api/bdaddr.rs

```diff
@@ -0,0 +1,270 @@
+//! Implementation of Bluetooth's MAC address.
+
+use std::fmt;
+use std::str::FromStr;
+use thiserror::Error;
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+#[cfg(feature = "serde")]
+use serde_cr as serde;
+
+use crate::Error;
+
+/// Stores the 6 byte address used to identify Bluetooth devices.
+#[cfg_attr(
+    feature = "serde",
+    derive(Serialize, Deserialize),
+    serde(crate = "serde_cr")
+)]
+#[derive(Copy, Clone, Hash, Eq, PartialEq, Default)]
+#[repr(C)]
```

Comment:
Is this type being used for FFI somewhere?

Reply:
That is legacy from rumble (?). I wouldn't mind removing it.